src/workerd/server/tests/container-client/test.js

```diff
@@ -197,6 +220,17 @@ export const testBasics = {
   },
 };
 
+// Test exit code monitor functionality
+export const testExitCode = {
+  async test(_ctrl, env) {
+    for (const CONTAINER of [env.MY_CONTAINER, env.MY_DUPLICATE_CONTAINER]) {
```

Comment:
I think it's ok to run this only on env.MY_CONTAINER. The test already takes 35 seconds and redundant tests significantly increases the test duration.